perf/istio-install/values.yaml

```diff
@@ -79,9 +79,9 @@ gateways:
       ver: ingress10
       app: istio-ingressgateway
       name: gateway
-    replicas: 3
-    autoscaleMin: 3
-    autoscaleMax: 5
+    replicas: 10
+    autoscaleMin: 10
```

Comment:
what's the meaning if replicacount and the hpa min and max are all the same number